doc/source/groupby.rst

```diff
@@ -62,16 +62,30 @@ See the :ref:`cookbook<cookbook.grouping>` for some advanced strategies.
 Splitting an object into groups
 -------------------------------
 
-pandas objects can be split on any of their axes. The abstract definition of
+Pandas objects can be split on any of their axes. The abstract definition of
```

Comment:
Can you revert this please? Besides that `pandas` as a brand is always lower case, we try to avoid unrelated changes in PRs, so in case there are problems, it's easier to see where they are coming from, revert only the failing commit without unrelated reversals...

Surely you can propose those changes in a separate PR, to improve the changes.

Reply:
This has been reverted, see b80ff177293437d0a306127e407ba8c671dd6b4f